spec-tests/src/test/java/org/candlepin/spec/ExportSpecTest.java

```diff
@@ -193,12 +212,48 @@ public void shouldImportContentAccessCertsForAConsumerBelongingToOwnerInSCAMode(
 
         assertThat(caCerts).singleElement();
 
-        // Check if entitlement certs are present in exported zip file.
+        // Should not contain entitlement certificate in SCA mode
         List<ZipEntry> entitlementCerts = export.stream()
             .filter(entry -> entry.getName().startsWith(ENTITILEMENT_CERTIFICATES_PATH))
             .filter(entry -> entry.getName().lastIndexOf('/') == ENTITILEMENT_CERTIFICATES_PATH.length() - 1)
             .collect(Collectors.toList());
 
+        assertThat(entitlementCerts).isEmpty();
+    }
+
+    @Test
+    public void shouldImportEntitlementCertsForAConsumerBelongingToOwnerInEntitlementMode() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = adminClient.owners().createOwner(Owners.random());
+        String ownerKey = owner.getKey();
+
+        ProductDTO modifiedProd = adminClient.ownerProducts().createProduct(ownerKey, Products.random());
+        ProductDTO prod = adminClient.ownerProducts().createProduct(ownerKey, Products.random());
+        ContentDTO cont = adminClient.ownerContent().createContent(ownerKey, Contents.random()
+                .modifiedProductIds(Set.of(modifiedProd.getId())));
+        adminClient.ownerProducts().addContentToProduct(ownerKey, prod.getId(), cont.getId(), true);
+        PoolDTO pool = adminClient.owners().createPool(ownerKey, Pools.random(prod));
+        ConsumerDTO consumer = adminClient.consumers().createConsumer(Consumers.random(owner));
+        ApiClient consumerClient = ApiClients.ssl(consumer);
+        consumerClient.consumers().bindPool(consumer.getUuid(), pool.getId(), 1);
+
+        File manifest = createCertExport(consumerClient, consumer.getUuid());
+        ZipFile export = ExportUtil.getExportArchive(manifest);
+
+        // Should not contain content access certs in exported zip file in Entitlement mode.
+        List<ZipEntry> caCerts = export.stream()
+                .filter(e -> e.getName().startsWith(CONTENT_ACCESS_CERTS_PATH))
+                .filter(e -> e.getName().lastIndexOf('/') == CONTENT_ACCESS_CERTS_PATH.length() - 1)
+                .collect(Collectors.toList());
+
+        assertThat(caCerts).isEmpty();
+
+        // Should contain entitlement certificate in Entitlement mode
+        List<ZipEntry> entitlementCerts = export.stream()
```

Comment:
Double indents here as well. Likely throughout all of these changes

Reply:
done